packages/indicators/src/Builder/helpers.ts

```diff
@@ -44,7 +44,7 @@ export const fetchAnalytics = async (
   return analytics;
 };
 
-export const evaluateFormulaToNumber = (
+export const convertBooleanToNumber = (
```

Comment:
Ah actually, I was thinking we'd remove the `evaluateToNumber(formula)` function from expression-parser all together. And we'd implement the same logic here instead? That function being called `evaluateToNumber` is super misleading I feel, and since it's only used here, lets just remove it all together 👍

Reply:
Nice suggestion👍 Have removed `evaluateToNumber()` in expression parser.